package.json

```diff
@@ -79,8 +79,8 @@
     "babel-core": "7.0.0-bridge.0",
     "bundlemon": "^1.0.0",
     "chrome-remote-interface": "0.27.1",
-    "cordova": "7.1.0",
-    "cordova-android": "7.1.4",
+    "cordova": "8.1.2",
```

Comment:
readme & docs to update?

Reply:
la doc c'est la PR https://github.com/cozy/cozy-drive/pull/2439